cmd/webserver.go

```diff
@@ -195,6 +199,14 @@ func getControlPlaneHandler(w http.ResponseWriter, r *http.Request) {
 	respondWithJSON(w, 200, resp)
 }
 
+func getActorsHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
though metadata returns actor related that can change later..so change this to getMetadataHandler